superset/utils/feature_flags.py

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=C,R,W
+from superset import feature_flags
```

Comment:
Are we sure there are no issues with circular imports here? For instance if I try to import this `feature_flags` here from `superset/__init__.py` or from say `superset/models/core.py`, are we sure it won't choke on a circular import?

In general I've been trying to keep everything in `utils/` not depend on anything in the superset package to avoid circular imports.

One workaround is to use `from flask import current_app`, and then reference `current_app.config`.

Reply:
Interesting question. I originally ran into a circular import issue by placing the utility function in `utils/core.py`. The second place I settled on was in `superset/__init__.py` itself as feature flags are tied to the app itself. I eventually moved it out into its own file to make testing easier but it is not strictly necessary for the `patch.dict` I was doing in tests.

I don't think we'll run into circular imports even if we import `is_feature_enabled` from `superset/models/core.py`, which imports from `superset`, because `superset` specifically imports `utils.core`, not all of `utils`.

How do you feel about placing this utility function in its own file VS in `superset/__init__.py`?